app/core/Router.coffee

```diff
@@ -13,7 +13,24 @@ module.exports = class CocoRouter extends Backbone.Router
     @initializeSocialMediaServices = _.once @initializeSocialMediaServices
 
   routes:
-    '': go('HomeView')
+    '': ->
+      # Testing new home page
+      group = me.get('testGroupNumber') % 4
+      group = 0
```

Comment:
Ah, need to remove that. Was leftover from a test. The test should be 'on' as soon as these changes are deployed.